files/image_config/warmboot-finalizer/finalize-warmboot.sh

```diff
@@ -101,16 +101,30 @@ function check_list()
     echo ${RET_LIST}
 }
 
+function set_cpufreq_governor() {
+    local -r governor="$1"
+    echo "$governor" | tee /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor 1> /dev/null \
+        && debug "Set CPUFreq scaling governor to $governor" \
+        || debug "Failed to set CPUFreq scaling governor to $governor"
+}
+
+function finalize_common() {
+    # Read default governor from kernel config
+    local -r default_governor=$(cat "/boot/config-$(uname -r)" | grep -o 'CONFIG_CPU_FREQ_DEFAULT_GOV_[^=]*=y')
```

Comment:
What's the guarantee that `"/boot/config-$(uname -r)"` will have the default governor setting?

Do you need to sanity check on the value and type of `default_governor` here? What if the `cat` or `grep` result in a failure? Wouldn't the code proceed to assign invalid value to `/sys/devices/system/cpu/cpu*/cpufreq/scaling_governor`

Reply:
It's part of our sonic linux kernel config, why would we remove it or set it to invalid value?